pinot-integration-tests/pom.xml

```diff
@@ -373,5 +381,10 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.hk2</groupId>
+      <artifactId>hk2-metadata-generator</artifactId>
```

Comment:
This module is needed in any project that wish its classes to be auto-discovered.
When building the project, this module will create `META-INF/hk2-locator/default` file in JAR, and write the class name into that filer properly.
Later the `ServiceAutoDiscoveryFeature` class will automatically load the class.